src/Pages/jiyeon/Login/Login.js

```diff
@@ -1,13 +1,78 @@
-import React, { Component } from 'react';
-
-class Login extends Component {
-    render() {
-        return (
-            <div>
-                
-            </div>
-        );
-    }
-}
+import React from "react";
+import { Link } from 'react-router-dom';
+import facebook_logo from '../../../images/jiyeon/facebook_icon.png'
+import './Login.scss';
+
+class LoginJY extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      idInput: '',
+      pwInput: '',
+      isBtnActive: false
+    };
+  }
+
+  handleIdPwInput = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
```

Comment:
계산된 속성명 잘 사용해주셨네요!! :) 
아래와 같이 구조분해할당도 해주시면 더욱 가독성에 좋은 코드가 될 것 같습니다.
```suggestion
    const { name, value } = event.target;
    this.setState({
      [name]: value
    });
```

Reply:
수정하겠습니다!